docs/superpowers/specs/2026-08-17-graphrag-readme-positioning-design.md

```
@@ -0,0 +1,425 @@
+    WHEN 'tentative' THEN 0
+    ELSE -1
+  END as rank_weight
+FROM walk w
+JOIN llm_wiki_entries n ON n.id = w.node_id
+GROUP BY n.id
+ORDER BY min_distance ASC, rank_weight DESC
+LIMIT /* maxNodes */;
+```
+
+> *Note: For clarity, the snippet above shows the final hydrated output in a single query. In the actual `expo-llm-wiki` codebase, the recursive CTE strictly calculates the `(node_id, distance)` graph traversal, and `GraphTraversalService` handles hydration in a subsequent optimized batch lookup.*
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)